spec.md

```diff
@@ -19,15 +19,242 @@
 
 ### Space Data Model
 
+`Space` properties:
+
+* `id` - deterministically set by server if not provided.
+* `type` - set by server to the string `Space`
+* `name` (optional)
+* `controller` - determined by server based on who creates the space originally
+
 #### Space JSON Representation
 
 #### Space CBOR Representation
 
-### Space Operations
+### Create Space operation
+
+To create a Space:
+
+* Perform an authenticated Create Space operation. The controller/invoker DID
```

Comment:
(not urgent)

can invoker be different than controller, but with a verifiable zcap delegation chain authorizing POST /spaces/ ?
(yes imho)

Reply:
Agreed, yes. We'll add that clarification when we explain delegations.

Reply:
I'll add a placeholder for that tho